python/sglang/bench_serving.py

```diff
@@ -594,6 +700,16 @@ def fire(args: argparse.Namespace):
         else f"http://{args.host}:{args.port}/v1/models"
     )
 
+    if args.backend == "trt":
+        api_url = (
+            f"{args.base_url}/v2/models/ensemble/generate_stream"
+            if args.base_url
+            else f"http://{args.host}:{args.port}/v2/models/ensemble/generate_stream"
+        )
+        if args.model is None:
```

Comment:
TensorRT LLM does not have a `/v1/models` API, thus `--model` needs to be **manually** specified.